ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2690,6 +2690,29 @@ periodics:
       resources:
         requests:
           cpu: 500m
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@daily'
+  decorate: true
+  name: periodic-openshift-release-qci-pruner
+  spec:
+    containers:
+    - args:
+      - --confirm=false
```

Comment:
We will make it to true when we are happy with the logs with `false`.

Reply:
This argument parser does not support an explicit `--confirm=false`. `--confirm` just needs to be omitted.

Reply:
Removed.